erpnext/setup/doctype/email_digest/email_digest.js

```diff
@@ -1,78 +1,31 @@
 // Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
 // License: GNU General Public License v3. See license.txt
 
-cur_frm.cscript.refresh = function(doc, dt, dn) {
-	doc = locals[dt][dn];
-	cur_frm.add_custom_button(__('View Now'), function() {
-		frappe.call({
-			method: 'erpnext.setup.doctype.email_digest.email_digest.get_digest_msg',
-			args: {
-				name: doc.name
-			},
-			callback: function(r) {
-				var d = new frappe.ui.Dialog({
-					title: __('Email Digest: ') + dn,
-					width: 800
-				});
-				$(d.body).html(r.message);
-				d.show();
-			}
+frappe.ui.form.on("Email Digest", {
+	refresh: function(frm) {
+		frm.add_custom_button(__('View Now'), function() {
```

Comment:
View Now should be visible only after save. So put thus inside the `!frm.is_new()` check below

Reply:
@marination Done.